language/move-lang/functional-tests/tests/diem/on_chain_config/disable_txes.move

```diff
@@ -0,0 +1,57 @@
+//! account: vivian, 1000000, 0, validator
+
+//! block-prologue
+//! proposer: vivian
+//! block-time: 2
+
+// disable txes from all accounts except DiemRoot
+//! new-transaction
+//! sender: diemroot
+script {
+use 0x1::DiemTransactionPublishingOption;
+fun main(account: &signer) {
+    DiemTransactionPublishingOption::halt_all_transactions(account);
+}
+}
+// check: "Keep(EXECUTED)"
+// check: NewEpochEvent
+
+// sending from normal account fails
+//! new-transaction
+script {
+fun main() {
+}
+}
+// check: "Discard(UNKNOWN_SCRIPT)"
```

Comment:
Could you use the `rotate_authentication_key` transaction here as well since it's in the allowlist (and is also the transaction that is used below to test that it's no longer halted). This way it's testing purely that transactions are halted, and not the allowlist as well possibly.